File: IOT/gurpreetk12_gurpreetkaur_2125cs1094_2/FunctionandArrays/BarChart.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++)
    {
        cin>>arr[i];
    }
    int max=arr[0];
    for(int i=0;i<n;i++)
    {
        if(arr[i]>=max)
        {
            max=arr[i];
        }
    }
    for(int i=max;i>0;i--)
    {
        for(int j=0;j<n;j++)
        {
            if(arr[j]>=i)
            {
                cout<<"*\t";
            }
            else
            {
                cout<<"\t";
            }
            
        }cout<<"\n";
    }
}

File: Web/Macabre908_AbhishekDubey_2125cse1175_2/FunctionAndArrays/barChart.java
import java.util.*;

public class barChart {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
    
        int max = 0;
        for(int i = 0; i < n; i++){
           arr[i] = sc.nextInt();
           max = Math.max(max, arr[i]);
        }
    
        for(int i = 0; i < max; i++){
           for(int val: arr){
             if(val >= max - i){
                System.out.print("*\t");
             } else {
                System.out.print("\t");
             }
           }
    
           System.out.println();
        }
        sc.close();
     }
}
